room/compiler-processing/src/test/java/androidx/room/compiler/processing/XRoundEnvTest.kt

```diff
@@ -222,6 +222,46 @@ class XRoundEnvTest {
         }
     }
 
+    @Test
+    fun getElementsFromPackageIncludesSources() {
+        val source = Source.kotlin(
+            "foo/Baz.kt",
+            """
+            package foo
+            class Baz 
+            """.trimIndent()
+        )
+
+        runProcessorTest(listOf(source)) { testInvocation ->
+            val elements = testInvocation.processingEnv.getTypeElementsFromPackage("foo")
+
+            val targetElement = testInvocation.processingEnv.requireTypeElement(
+                "foo.Baz"
+            )
+
+            assertThat(
+                elements
+            ).contains(targetElement)
+        }
+    }
+
+    @Test
+    fun getElementsFromPackageIncludesBinaries() {
+        runProcessorTest { testInvocation ->
+            val kspElements = testInvocation.processingEnv.getTypeElementsFromPackage(
```

Comment:
nice hack :D i usually compile a library in test to achieve this but this is good enough.